TriagerXChatbot/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Aqa-test-tools repo is using Apache license (ref https://github.com/adoptium/aqa-test-tools/blob/master/LICENSE), we can either remove this file, or change it to Apache license. Thanks.

Reply:
Resolved